Jenkinsfile

```diff
@@ -152,8 +152,11 @@ timestamps {
         notifyBuildFailure channel: channel
         throw err
       } finally {
+        def customDataMap = [:]
+        customDataMap['branchName'] = env.BRANCH_NAME
```

Comment:
A `env.BRANCH_NAME` will be different for every pull request, is that alright? Will Grafana be configured to present all builds where branch name starts with `PR-` on single dashboard? Is such dashboard already configured in there?

Reply:
What i'm trying to achieve with this new addition is a way to separate the master branch from the PR as at the moment, as you can see in grafana, when you try to calculate the average build time, or health and so on it puts both PR and master together.
 
Below you can find the link to the grafana Dashboard.

https://grafana.reform.hmcts.net/dashboard/db/build-dashboard?orgId=1&var-project_name=CMC%20Citizen%20Frontend